stf/feature_engineering/feature_free_days.py

```diff
@@ -6,11 +6,13 @@
 
 import numpy as np
 import pandas as pd
+from datetime import date
+import holidays
 
 HOLIDAY_CSV_PATH = os.path.dirname(__file__) + "/dutch_holidays_2020-2022.csv"
 
 
-def create_holiday_functions():
+def create_holiday_functions(country="FR", years=None):
     """
     This function provides functions for creating holiday feature.
     This improves forecast accuracy. Examples of features that are added are:
```

Comment:
Still work in progress, but don't forget to document the arguments. Probably nice to refer to the country code standard that is used and so on.